### ISSUE TITLE
Unable to download complete blockchain/ use wallet

### ISSUE THREAD

**Issue by tommyquite** [ethereum/wiki#414]
Hi,

I am new to ethereum so I apologize for any dumb questions. I have downloaded the original ethereum wallet and when I openened it, it started downloading the blockchain. Had 7 peers and got all the way to 61% of bloacks after a few hours. Then it got stuck. 

Now everytime I re-try it by closing and opening the wallet it gets stuck early on. Right away or after 1-2%. And most of the times under 6 peers.

The the meantime I have sent some Ether from my Poloniex exchange account to my Ethereum wallet adress. However it does not appear in my account. 

2 questions.
1. Where is the ether that I have sent to my wallet? Can I only see it or receive it after the full blockchain is downloaded? And if so, will it appear once the full blockchain is downloaded?

2. How can I download the full blockchain?

Thanks in advance.

Bets regards,

Tom

**Comment by ONslaughTT** [comment]
I would like to know if there is a safe link to download the blockchain so I could import it, I am having syncing problems too.

**Comment by EssenceOfChaos** [comment]
I tried to download the blockchain for days. After 4 days it finally downloaded but I still couldn't see the recent transactions I sent to my account but so I'm assuming it never fully caught up. 

Then the latest version of the wallet came out and I downloaded it and now it looks like it is going to try to resync everything it's at 16k blocks out of 3.6 million... but last week I was nearly synced up to the blockchain so I don't understand what's happening.

Is there a light wallet that lets you sync from, for example, 1 month or 2 months prior? This would be ideal, rather than having to sync from the genesis block.
